### ISSUE TITLE
`aria-controls` renders as "undefined" in props table

### ISSUE THREAD

**Issue by taion** [react-bootstrap/react-bootstrap#1370]
![image](https://cloud.githubusercontent.com/assets/3112159/10239729/f0a1284c-689f-11e5-8b22-9722e69b7737.png)

That last prop there should be `aria-controls`.

**Comment by AlexKVal** [comment]
I've found the culprit. PR is on the way.
Though it will go to https://github.com/jquense/react-component-metadata

I close this in favor of https://github.com/jquense/react-component-metadata/issues/1

**Comment by AlexKVal** [comment]
https://github.com/jquense/react-component-metadata/pull/2
